### DISCUSSION TITLE
raccogliere tutti i progetti realizzati a partire da questo repo

### DISCUSSION THREAD

**Discussion by pigreco** [opendatasicilia/ods-mkdocs-material#18] [Idee/Proposte]
creare un file md e annotare tutti i link ai progetti in oggetto

qui un esempio

https://hfcqgis.opendatasicilia.it/

**Comment by gbvitrano** [comment]
https://opendatasicilia.github.io/OpenDataSicilia-per-il-Coronavirus/
